common/src/main/java/com/wynntils/core/functions/Function.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.functions;
+
+public interface Function {
+    String getValue(String argument);
+
+    String getName();
+
+    default void init() {}
```

Comment:
I assume we are waiting on the manager pr for dependencies

Reply:
Yeah, maybe. We'll have to see how we sort that out. At the moment, I assume functions has no special dependencies as such. There is an `EnablableFunction` which can request stuff to initialize and/or check for the presense of prerequisites. Let's build on top of that when the manager PR has been sorted out.